src/Microsoft.OpenApi/Validations/Rules/OpenApiResponseRules.cs

```diff
@@ -16,17 +16,16 @@ public static class OpenApiResponseRules
         /// <summary>
         /// Validate the field is required.
         /// </summary>
-        public static ValidationRule<OpenApiResponse> FieldIsRequired =>
+        public static ValidationRule<OpenApiResponse> ResponseRequiredFields =>
             new ValidationRule<OpenApiResponse>(
                 (context, response) =>
                 {
                     // description
                     context.Enter("description");
                     if (String.IsNullOrEmpty(response.Description))
```

Comment:
>String.IsNullOrEmpty(response.Description)) [](start = 24, length = 43)

Based on your response in [https://github.com/Microsoft/OpenAPI.NET/issues/223](https://github.com/Microsoft/OpenAPI.NET/issues/223), this should only check for null at the moment, right?

Reply:
Yeah, I hadn't fixed all the cases.  I should have before pushing the PR, but I ran out of time and wanted to get feedback on the overall set of changes.